pull_request_template.md

```diff
@@ -1,25 +1,36 @@
 <!-- First of all, THANK YOU for your contribution. -->
 
 ## Changes Proposed:
--  
--  
+-  [ ] This pull request promotes changes to the core (units, players, creatures, game systems).
+-  [ ] This pull request promotes changes to scripts (bosses, spell scripts, creature scripts).
+-  [ ] This pull request promotes database changes.
```

Comment:
it would be nice to clarify the difference between "core" and "scripts" as I see often a lot of confusion. Something like:

```suggestion
-  [ ] This pull request promotes changes to _SCRIPTS_  - any **C++** changes under `src/server/scripts` (e.g. bosses, spell scripts, creature scripts).
-  [ ] This pull request promotes changes to the _CORE_  - any **C++** changes which are NOT Scripts (e.g. scripts units, players, creatures, game systems).
-  [ ] This pull request promotes _DATABASE_ changes - any **SQL** changes.
```